src/hotspot/share/jfr/metadata/metadata.xml

```diff
@@ -156,6 +156,12 @@
     <Field type="ulong" name="id" label="Continuation ID" />
   </Event>
 
+  <Event name="VirtualThreadPinned" category="Java Virtual Machine, Runtime" label="Virtual Thread Pinned" thread="true" stackTrace="true">
+    <Field type="string" name="blockingOperation" label="Blocking operation" />
```

Comment:
Previously, the event was in the "Java Application" category. I think that was a better fit because it meant it was visualized in the same lane in a thread graph. See here for more information about the category:

https://docs.oracle.com/en/java/javase/21/docs/api/jdk.jfr/jdk/jfr/Category.html

(Note: The fact that the event is now written in the JVM doesn't determine the category.)

Reply:
Thanks for spotting this, it wasn't intended to change the category. I think it's that Event element was copied from another event when adding it to metadata.xml and value that was in the `@Catalog` wasn't carried over.